presto-main/src/main/java/com/facebook/presto/metadata/FunctionManager.java

```diff
@@ -343,6 +347,15 @@ private FunctionHandle lookupFunction(
         throw new PrestoException(FUNCTION_NOT_FOUND, constructFunctionNotFoundErrorMessage(functionName, parameterTypes, candidates));
     }
 
+    private FunctionNamespaceManager<?> getRequiredServingFunctionNamespaceManager(CatalogSchemaName functionNamespace)
+    {
+        Optional<FunctionNamespaceManager<?>> functionNamespaceManager = getServingFunctionNamespaceManager(functionNamespace);
+        if (!functionNamespaceManager.isPresent()) {
+            throw new PrestoException(FUNCTION_NOT_FOUND, format("Cannot find function namespace '%s'", functionNamespace));
```

Comment:
If this is called during `CREATE FUNCTION` then this error code is not appropriate. Maybe this class should just throw run time error and let caller decide what error code it should use.

Reply:
Since different callers need to handle them differently, I'm removing this method and handle the cases of `!functionNamespaceManager.isPresent()` in each caller (2) separately.